WindowsCompatibility/WindowsCompatibility.psm1

```diff
@@ -701,4 +701,8 @@ function Add-WindowsPSModulePath
         $env:PSModulePath += ";${env:userprofile}\Documents\WindowsPowerShell\Modules;${WindowsPSModulePath}"
     }
 
+    if (-not $env:PSModulePath.Contains('C:\Program Files\WindowsPowerShell\Modules'))
```

Comment:
What I would suggest is checking for the paths individually and adding them if they don't exist. Also, need to use the env vars and not `c:\Program Files`

something like this:

```powershell
    $WindowsPSModulePath = [System.Environment]::GetEnvironmentVariable("psmodulepath", [System.EnvironmentVariableTarget]::Machine)
    if (-not ($env:PSModulePath).Contains("${env:userprofile}\Documents\WindowsPowerShell\Modules"))
    {
        $env:PSModulePath += ";${env:userprofile}\Documents\WindowsPowerShell\Modules"
    }
    if (-not  ($env:PSModulePath).Contains("${env:programfiles}\WindowsPowerShell\Modules"))
    {
        $env:PSModulePath += ";${env:programfiles}\WindowsPowerShell\Modules"
    }
    if (-not ($env:PSModulePath).Contains($WindowsPSModulePath))
    {
        $env:PSModulePath += ";${WindowsPSModulePath}"
    }
```

Reply:
May be simpler to add it, then split the path into an array and remove the duplicates

```powershell
[string]::Join(";",(("a;b;b;c").Split(";") | Sort-Object -Unique))
```

Reply:
@SteveL-MSFT Have a look at the changes since this piece where you commented.